src/main/java/survey/Survey.java

```diff
@@ -39,35 +40,36 @@ public Survey(String name, Collection<Question> questions){
     }
 
     /*
-     * Sets the name
-     * @param name {String}
+     * Sets the id
+     * @param id {int}
      */
-    public void setName(String name) {
-        this.name = name;
+    public void setId(Integer id) {
```

Comment:
shouldn't this method call super? Is the id field only being changed for subclasses, not parent class? (protected method)?

Reply:
Survey doesn't have a parent class with an integer id field. The Object class doesn't have a setId https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html.